src/opt/example/KnapsackEvaluationFunction.java

```diff
@@ -12,60 +12,61 @@
 public class KnapsackEvaluationFunction implements EvaluationFunction {
     
     /**
-     * The weights for the things that can be put in the sack

```

Comment:
In this file, I made the terms "weight" and "value" consistent. Before these changes it was using "weights" and "volumes" in a confusing way.